portfolio/src/main/webapp/index.html

```diff
@@ -5,21 +5,35 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+    body {
+      background-image: url('https://i.pinimg.com/originals/92/17/4a/92174ad3758fdb8f9c2e4579ad5430a0.jpg');
+      background-size: cover;
+    }
+    </style> -->
```

Comment:
Are you planning to use this? Generally we do not commit code that has been commented out as it will quickly get stale and may not even work.